dist_tables/querying.rst

```diff
@@ -91,13 +91,13 @@ Broadcast joins
 
 This join type is used while joining small tables with each other or with a large table. This is a very common use case where you want to join the keys in the fact tables (large table) with their corresponding dimension tables (small tables). Citus replicates the small table to all workers where the large table's shards are present. Then, all the joins are performed locally on the workers in parallel. Subsequent join queries that involve the small table then use these cached shards.
 
-Colocated joins
+Co-located joins
```

Comment:
Yep I replaced the non-hyphenated versions through the whole docs except where they refer to UDFs or filenames. Glad you asked just now though because after another search I found two isolated instances I missed.